.github/workflows/docker_build_base.yml

```diff
@@ -0,0 +1,261 @@
+name: Docker Build Base
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+
+concurrency:
+  group: format('{0}-{1}', ${{ github.ref }}, 'Docker Build Base')
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  build_and_push_base:
+    name: Build and push vitess/base Docker images
+    runs-on: gh-hosted-runners-16cores-1
+    if: github.repository == 'vitessio/vitess'
+
+    strategy:
+      fail-fast: true
+      matrix:
+        branch: [ latest, mysql57, percona57, percona80 ]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Set Dockerfile path
+        run: |
+          if [[ "${{ matrix.branch }}" == "latest" ]]; then
+            echo "DOCKERFILE=./docker/base/Dockerfile" >> $GITHUB_ENV
+          else
+            echo "DOCKERFILE=./docker/base/Dockerfile.${{ matrix.branch }}" >> $GITHUB_ENV
+          fi
+
+      - name: Build and push on main
+        if: github.ref == 'refs/heads/main'
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ${{ env.DOCKERFILE }}
+          push: true
+          tags: vitess/base:${{ matrix.branch }}
+
+      ######
+      # All code below only applies to new tags
+      ######
+
+      - name: Get the Git tag
+        if: startsWith(github.ref, 'refs/tags/')
+        run: echo "TAG_NAME=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+
+      - name: Set Docker tag name
+        if: startsWith(github.ref, 'refs/tags/') && matrix.branch == 'latest'
```

Comment:
For `vitess/base` on a new tag, we only build `vitess/base:vX.X.X`, using the default Dockerfile, we ignore all other flavor of MySQL/Percona